python/rikai/spark/sql/codegen/mlflow_registry.py

```diff
@@ -215,6 +215,8 @@ def resolve(self, raw_spec):
         uri = raw_spec.getUri()
         logger.info(f"Resolving model {name} from {uri}")
         parsed = urlparse(uri)
+        if parsed.netloc:
+            raise ValueError("URI with 2 forward slashes is not supported")
```

Comment:
The error message should tell the user how to do the right thing. e.g., you might want to add "Try <user input but with 1 slash> or <user input but with 3 slashes> instead" to the message

Reply:
Improved.